README.md

```diff
@@ -73,7 +73,33 @@ npm run system-test
 Prior to sending any data you can run the following docker command to log each query to the console to examine each query. No data is sent to the destination using this command.
 
 ```sh
-docker run -i -e ASTRA_CLOUD_USERNAME=<your_username> -e ASTRA_CLOUD_PASSWORD=<your_password> adastradev/data-ingestion-agent:latest preview
+docker run -i \
+-e ASTRA_CLOUD_USERNAME=<your_username> \
+-e ASTRA_CLOUD_PASSWORD=<your_password> \
+adastradev/data-ingestion-agent:latest \
+preview
+```
+
+## Adhoc Ingestion
+To immediately begin the ingestion process you can run the following with the 'ingest' flag. This command will terminate the container once the process has completed.
+
+```sh
+docker run -i \
+-e ASTRA_CLOUD_USERNAME=<your_username> \
+-e ASTRA_CLOUD_PASSWORD=<your_password> \
+-e ORACLE_ENDPOINT=hostname:port/service_name \
+-e ORACLE_USER=user \
+-e ORACLE_PASSWORD=password \
+adastradev/data-ingestion-agent:latest \
+ingest
+```
+
+## Root Access to a running agent container
+
+After starting the agent and confirming a healthy status you can use the containers name or ID to access the virtual machine via command line (bash) as follows:
+
+```sh
+docker exec -it <container_id_or_name> /bin/bash
```

Comment:
Literally plops you in as root 

e.g. root@0421d255faf9:/app#

Reply:
We might check with Susan to make sure this meets her expectations. It allows root access w/ a new run of the agent. I'm not sure if it's possible to get root access to an already-running container.

Reply:
This works on an existing running container where as we'd need to do some potentially intrusive changes to the dockerfile to set things up for hopping in as root from a singl docker run command